src/app/shared/layout/header.component.ts

```diff
@@ -163,6 +163,10 @@ export class HeaderComponent implements OnInit {
   }
 
   viewBrowser() {
+    console.log(this.filtersService.getEncodedFilter());
+    console.log(this.githubService.getRepoURL());
```

Comment:
`console.log` is a bit informal and less polished. 
To standardize logging, let's switch to using `logger: LoggingService` instead to log this instead or remove this all together!